dev/io.openliberty.microprofile.openapi.4.0.internal.services/bnd.bnd

```diff
@@ -35,17 +35,18 @@ Private-Package: io.openliberty.microprofile.openapi40.internal.services.impl
 WS-TraceGroup: MPOPENAPI
 
 -buildpath: \
-    io.openliberty.io.smallrye.openapi4.core;version=latest,\
-    io.openliberty.io.smallrye.openapi4.jaxrs;version=latest,\
-    io.openliberty.io.smallrye.jandex3;version=latest,\
-    io.openliberty.microprofile.openapi.2.0.internal;version=latest,\
-    io.openliberty.org.eclipse.microprofile.openapi.4.0,\
-    com.ibm.ws.adaptable.module;version=latest,\
-    com.ibm.ws.logging;version=latest,\
-    com.ibm.websphere.org.osgi.core;version=latest,\
-    com.ibm.websphere.org.osgi.service.component;version=latest,\
-    com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
-    com.ibm.ws.org.osgi.annotation.versioning;version=latest
+	io.openliberty.io.smallrye.openapi4.core;version=latest,\
+	io.openliberty.io.smallrye.openapi4.jaxrs;version=latest,\
+	io.openliberty.io.smallrye.jandex3;version=latest,\
+	io.openliberty.microprofile.openapi.2.0.internal;version=latest,\
+	io.openliberty.org.eclipse.microprofile.openapi.4.0,\
+	com.ibm.ws.adaptable.module;version=latest,\
+	com.ibm.ws.logging;version=latest,\
+	com.ibm.websphere.org.osgi.core;version=latest,\
+	com.ibm.websphere.org.osgi.service.component;version=latest,\
+	com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
+	com.ibm.ws.org.osgi.annotation.versioning;version=latest,\
+	com.ibm.ws.kernel.boot.core
```

Comment:
I'm unsure why this change was needed, but I don't see it causing any harm.

Reply:
Oh I know what happened. That was for the other PR to `import com.ibm.ws.kernel.productinfo.ProductInfo;` and entered the wrong commit.